tools/plotjuggler/juggle.py

```diff
@@ -76,10 +76,13 @@ def process(can, lr):
   return [d for d in lr if can or d.which() not in ['can', 'sendcan']]
 
 
-def juggle_route(route_or_segment_name, can, layout, dbc=None):
+def juggle_route(route_or_segment_name, can, layout, dbc, should_migrate):
   sr = LogReader(route_or_segment_name, default_mode=ReadMode.AUTO_INTERACTIVE)
 
   all_data = sr.run_across_segments(24, partial(process, can))
+  if should_migrate:
+    from openpilot.selfdrive.test.process_replay.migration import migrate_all
```

Comment:
why late import?

Reply:
i thought you could run tools without building openpilot (which is required for migration.py imports), so i wanted to keep that, but now i see that doesn't work